### PR TITLE
Feature/tags configs cleanup

### PR DESCRIPTION
**Pull request by vdektiarev**
1. Removed internal error topic arn output from alerting stack (can't be two same outputs in the same app)
2. Passing tags to the stacks instead of tags applying commands duplication
3. Refactored monitored stack code to extract resources creation into methods
4. Adapted stacks to config file changes
5. Removed env admin email from sample configs, as we won't use them

### CODE REVIEW

**Review by natmasslove — APPROVED** [review]

File: infra_tooling_account/app.py [hunk 1]
```diff
@@ -18,12 +18,14 @@
...
     app,
     f"cf-{PROJECT_NAME}-InfraToolingCommonStack-{STAGE_NAME}",
+    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
     stage_name=STAGE_NAME,
     project_name=PROJECT_NAME,
 )
 InfraToolingAlertingStack(
     app,
     f"cf-{PROJECT_NAME}-InfraToolingAlertingStack-{STAGE_NAME}",
+    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
```
Comment (natmasslove) [inline infra_tooling_account/app.py:28]: Alerting stack now relies on Common stack's output, so we need to make sure Alerting is created ONLY after Common is completed (I bumped into problem like this).

commonStack = InfraToolingCommonStack(
    app,
    f"cf-{PROJECT_NAME}-InfraToolingCommonStack-{STAGE_NAME}",
    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
    stage_name=STAGE_NAME,
    project_name=PROJECT_NAME,
)
infraStack = InfraToolingAlertingStack(
    app,
    f"cf-{PROJECT_NAME}-InfraToolingAlertingStack-{STAGE_NAME}",
    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
    stage_name=STAGE_NAME,
    project_name=PROJECT_NAME,
)

infraStack.add_dependency(commonStack)

File: infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py
Comment (natmasslove) [inline infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py:1]: Another small issue - eventbusrule-salmon-alerting-lambda-{stage} is not getting tagged.
I guess, we need to employ custom tagging for this specific resource

Comment (natmasslove) [reply infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py:1]: Looks like CloudFormation's shortcoming. No actions needed.
https://github.com/aws/aws-cdk/issues/4907

File: infra_tooling_account/app.py [hunk 2]
```diff
@@ -18,12 +18,14 @@
 InfraToolingCommonStack(
     app,
     f"cf-{PROJECT_NAME}-InfraToolingCommonStack-{STAGE_NAME}",
+    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
```
Comment (natmasslove) [inline infra_tooling_account/app.py:21]: Maybe, it's better to define tags var once
tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME}
and use it in all stack's params?

File: infra_tooling_account/app.py (hunk 1 above)
Comment (vdektiarev) [reply infra_tooling_account/app.py]: Done

File: infra_tooling_account/app.py (hunk 2 above)
Comment (vdektiarev) [reply infra_tooling_account/app.py]: Done

**Pull request merged**